backend/src/migrations.rs

```diff
@@ -266,15 +266,17 @@ pub enum SchemaVersion {
     CreatePltTokenAndEventTables,
     #[display("0040: Alter PLT events add event_timestamp and index")]
     AlterPltEventsAddEventTimestampAndIndex,
+    #[display("0041: m0041_fix_token_module_reject_reasons")]
```

Comment:
be consistent and write a proper message for display in line with the other displays